postgres/src/copy_out_reader.rs

```diff
@@ -46,7 +46,7 @@ impl BufRead for CopyOutReader<'_> {
             };
         }
 
-        Ok(self.cur.bytes())
+        Ok(self.cur.chunk())
```

Comment:
I think this can just be `Ok(&self.cur)`.

Reply:
Done.